src/components/preview-icon/PreviewIcon.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { PreviewIcon } from './PreviewIcon';
+import { IconStarOutlined } from '../../icons/IconStarOutlined';
+
+export default {
+  title: `Components/PreviewIcon`,
+  component: PreviewIconDemo,
+  argTypes: {
+    size: {
+      name: "size",
+      description: "Size of the preview icon",
+      control: { type: "number" },
+      type: "number"
+    },
+  },
+};
+
+function PreviewIconDemo(args) {
```

Comment:
This component is repeated and not actually needed, let's discuss on why the original is not being used.